spinalcordtoolbox/deepseg/models.py

```diff
@@ -0,0 +1,46 @@
+# coding: utf-8
+# List of available models.
+
+
+import os
+
+from spinalcordtoolbox import __sct_dir__
+
+
+MODELS = {
+    'cord-t2star': {'url': 'https://osf.io/8nsa6/',
+                    'short_description': 'Cord segmentation on T2*-weighted contrast.',
+                    'long_description': 'TODO'},
+    'cord-mouse': {'url': 'URL',
+                   'short_description': 'Cord segmentation on mouse MRI.',
+                   'long_description': 'Long description'}
+    }
+
+
+class DeepsegModel:
+    """
+    Main class that deals with deep learning models. When initialized, the class will inherit the subfields from
+    MODELS[modelx] dict.
+    """
+    def __init__(self, name_model):
+        self.name = name_model
+        self.folder = os.path.join(__sct_dir__, 'models', name_model)
+        for key in MODELS[name_model]:
+            setattr(self, key, MODELS[name_model][key])
+
+    def is_installed(self):
+        """
+        Check if model (.pt file) is installed under SCT directory
+        """
+        if os.path.exists(os.path.join(self.folder, self.name + '.pt')):
```

Comment:
Should we test if the `.json` is also present? or could we reasonably assume that?

Reply:
i was thinking about that. My reasoning here was that the json file is not "mandatory" to execute a segmentation. Only the model is, which is why I didn't check for it. Open for discussion.

Reply:
> My reasoning here was that the json file is not "mandatory" to execute a segmentation.

ah! actually i am wrong. It **is** mandatory! See: https://github.com/neuropoly/ivado-medical-imaging/blob/master/ivadomed/utils.py#L624.

Oh well, I will check for it then ;-)

Reply:
Done in b469daabba160a8f62827f5696a6e8a666a23dd5

Reply:
Great !